### PR TITLE
fix restore.sh

### PR DESCRIPTION
**Pull request by skloss**

### CODE REVIEW

**Comment by skloss** [comment]
CLA signature registration done

File: contrib/cluster-backup/restore.sh [hunk 1]
```diff
@@ -141,7 +141,7 @@ main() {
     for ns in ${NAMESPACES}; do
         if [ -e "${TEMP_DIR}/${ns}" ] && [ -d "${TEMP_DIR}/${ns}" ]; then
         echo "Restoring resources for '${ns}' Namespace..."
-        kubectl create -f <(echo $(cat ${TEMP_DIR}/namespaces.json | jq -r --arg NS "$ns" 'select( .items[].metadata.name | contains($NS))'))
+        kubectl create -f <(echo $(cat ${TEMP_DIR}/namespaces.json | jq -r --arg NS "$ns" '.items[] | select(.metadata.name | contains($NS))'))
```
Comment (mumoshu) [inline contrib/cluster-backup/restore.sh]: @skloss Wow! So, it was (mistakenly) restoring all the namespaces at once regardless of what was said to be restored?

Comment (skloss) [reply contrib/cluster-backup/restore.sh]: @mumoshu  Yes,exactly and also it wasn't creating any ressource inside the namespace

Comment (skloss) [reply contrib/cluster-backup/restore.sh]: tested it also on two cluster's

**Comment by skloss** [comment]
fixed bug: if the namespace was containing the namespace of other the script failed at line 142, thats why i replaced the contains with equals.

**Comment by mumoshu** [comment]
@skloss Good catch! Thanks a lot for the fix 🎉

**Pull request merged**